CODEOWNERS

```diff
@@ -6,6 +6,7 @@ packages/eslint-config-cozy-app @cpatchane
 packages/commitlint-config-cozy @kosssi @enguerran
 packages/browserslist-config-cozy @cpatchane @Crash--
 packages/cozy-app-publish @cpatchane @gregorylegarec @y-lohse
+packages/cozy-ci-utils @y-lohse
```

Comment:
Maybe add another one here to not be alone maintaining this package, if no one you can add me.

Reply:
Bah, I don't mind :) Some other packages in the list have only one maintainer too!

Reply:
As you wish